torch/csrc/jit/passes/utils/memory_dag.cpp

```diff
@@ -1,12 +1,28 @@
 #include "memory_dag.h"
 
 #include <torch/csrc/utils/memory.h>
+#include <c10/util/flat_hash_map.h>
 #include <algorithm>
 #include <queue>
 
 namespace torch {
 namespace jit {
+ska::flat_hash_map<const Element*, int> comprMap;
```

Comment:
Can we use `unsigned` instead of `int` to match the type of SparseBitVector's index